gdpr/impl.go

```diff
@@ -74,7 +79,16 @@ func (p *permissionsImpl) PersonalInfoAllowed(ctx context.Context, bidder openrt
 	}
 
 	if id, ok := p.vendorIDs[bidder]; ok {
-		return p.allowPI(ctx, id, consent)
+		basicEnforce := false
+		if account != nil {
+			for _, vendor := range account.GDPR.BasicEnforcementVendors {
+				if vendor == string(bidder) {
+					basicEnforce = true
+					break
+				}
+			}
+		}
+		return p.allowPI(ctx, id, consent, basicEnforce)
```

Comment:
I recommend passing in a `basicEnforce` flag as a parameter and moving the vendor lookup logic up a level. This will make it easier for us to add a global level basic mode.

Reply:
Seems reasonable.